packages/wallets/glow/src/adapter.ts

```diff
@@ -22,12 +22,14 @@ interface GlowWalletEvents {
     disconnect(...args: unknown[]): unknown;
 }
 
+type Network = 'mainnet' | 'devnet'
```

Comment:
Could you use the `WalletAdapterNetwork` type here?

https://github.com/solana-labs/wallet-adapter/blob/a827686dbc109ef43b5da0031811cb5013aaaac8/packages/core/base/src/types.ts#L6-L10

Ideally this type should be compatible with the other adapters.

Reply:
Will update 👍